sample-applications/blazing-coffee/BlazingCoffee/Client/Pages/SalesReports/SalesByDateChart.razor

```diff
@@ -73,4 +102,57 @@
 
         return date.ToString("yyyy-MMM");
     }
+
+    protected override async Task OnInitializedAsync()
+    {
+        await GetChartData();
+    }
+
+    async Task GetChartData()
+    {
+        Data = await Http.GetFromJsonAsync<SalesByDateViewModel[]>($"api/sales/RegionalSalesByDate/{StartValue?.ToString("s", CultureInfo.InvariantCulture)}/{EndValue?.ToString("s", CultureInfo.InvariantCulture)}");
```

Comment:
```suggestion
        var startValueParameter= $"{StartValue?.ToString("s", CultureInfo.InvariantCulture)}";
        var endValueParameter = $"{EndValue?.ToString("s", CultureInfo.InvariantCulture)";
        var url = $"api/sales/RegionalSalesByDate/{startValueParameter}/{endValueParameter}";
        Data = await Http.GetFromJsonAsync<SalesByDateViewModel[]>(url);
```

The URL parts can be split so that it is easier to read and understand it.